larky/src/main/resources/vendor/Crypto/Cipher/_mode_cfb.star

```diff
@@ -278,13 +248,18 @@ def _create_cfb_cipher(factory, **kwargs):
         iv = IV
 
     if len(iv) != factory.block_size:
-        fail(" ValueError(\"Incorrect IV length (it must be %d bytes long)\" %\n                factory.block_size)")
+        fail("ValueError: Incorrect IV length (it must be %d bytes long)" % factory.block_size)
 
     segment_size_bytes, rem = divmod(kwargs.pop("segment_size", 8), 8)
     if segment_size_bytes == 0 or rem != 0:
         fail(" ValueError(\"'segment_size' must be positive and multiple of 8 bits\")")
 
     if kwargs:
-        fail(" TypeError(\"Unknown parameters for CFB: %s\" % str(kwargs))")
-    return CfbMode(cipher_state, iv, segment_size_bytes)
+        fail("TypeError: Unknown parameters for CFB: %s" % str(kwargs))
+
+    return _CfbMode(cipher_state, iv, segment_size_bytes)
+
+CfbMode = larky.struct(
+    _create_cfb_cipher=_create_cfb_cipher,
```

Comment:
You need to add a `__name__` here as well

Reply:
done